project/Dependencies.scala

```diff
@@ -108,5 +121,5 @@ object Dependencies {
     workbenchGoogleTests,
     workbenchMetrics,
     sam
-  )
+  ) ++ circe
```

Comment:
I'm not sure I like breaking convention like this, but I also like how the dependencies are grouped together and the version is listed once. (That came straight from the circe docs.) Any suggestions for improvements? My only thought just now is putting `circe` first, i.e.:
```
val rootDependencies = circe ++ Seq(
  ...
)
```
Maybe there are other groupings that could be treated the same way.